### PR TITLE
Bug/redirectplaytohome

### PR DESCRIPTION
**Pull request by Billwilke42**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [ ] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type
 
 
Please check the type of change your PR introduces:
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [x] Testing
- [ ] Other (please describe): 

## What is the current behavior?

- If /play is rendered without any data it triggers an error. The loading page has no animation. No thunk written for microservice. Errors in tests.

## What is the new behavior?

- /play will redirect to / if no data is available. Loading page has animation. New thunk written for the microservice. 

## Does this introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: src/App/App.js [hunk 1]
```diff
@@ -21,7 +20,7 @@ class App extends React.Component {
     if(this.props.isLoading) {
       return (
         <main className='App'>
-          <p className='loading'>Loading...</p>
+          <p className='loading'>Loading<span>.</span><span>.</span><span>.</span></p>
```
Comment (KyleWong2510) [inline src/App/App.js:23]: this is so cool!

File: src/App/App.css [hunk 2]
```diff
@@ -10,4 +10,32 @@
...
+
+.loading span:nth-child(2) {
+  animation-delay: .2s;
+}
+
+.loading span:nth-child(3) {
+  animation-delay: .4s;
+}
+
+@keyframes blink {
```
Comment (lilianaweimer) [inline src/App/App.css:31]: I love the loading animation!

File: src/Chalkboard/Chalkboard.js [hunk 3]
```diff
@@ -69,6 +76,12 @@ class Chalkboard extends React.Component {
   }
 
   render() {
+    if (this.props.questions.length === 0) {
+      return (
+        <Redirect to='/'/>
```
Comment (lilianaweimer) [inline src/Chalkboard/Chalkboard.js:81]: Really happy we got this working!

File: src/Chalkboard/Chalkboard.js [hunk 4]
```diff
@@ -22,12 +23,18 @@ class Chalkboard extends React.Component {
...
   }
+
+  componentDidMount() {
+  if(this.props.questions.length) {
+    this.setState({question: this.props.questions[this.props.currentRound][this.props.currentQuestion]})
+    }
+  }
   
   checkAnswer = (e) => {
     this.setState((prevState) => {
```
Comment (KyleWong2510) [inline src/Chalkboard/Chalkboard.js:40]: What is this checkAnswer doing now?

File: src/Lives/Lives.test.js [hunk 5]
```diff
@@ -17,20 +17,15 @@ describe('Lives', () => {
 		const { getByText, getAllByTestId } = render(
 				<BrowserRouter>
 					<Provider store={store}>
-            <Header 
-              question={{category: 'test category'}}
-              questionCounter={1} 
+            <Lives
```
Comment (lilianaweimer) [inline src/Lives/Lives.test.js:20]: The indentation got wacky here again :(

File: src/Chalkboard/Chalkboard.js [hunk 6]
```diff
@@ -69,6 +76,12 @@ class Chalkboard extends React.Component {
   }
 
   render() {
+    if (this.props.questions.length === 0) {
+      return (
+        <Redirect to='/'/>
+      )
+    }
+    const question = this.props.questions[this.props.currentRound][this.props.currentQuestion]
```
Comment (KyleWong2510) [inline src/Chalkboard/Chalkboard.js:84]: nice...cleans things up a lot more

**Pull request merged**